subprojects/freetype1.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = freetype-1.4
+source_url = https://sourceforge.net/projects/gnuwin32/files/freetype/1.4/libttf-1.4-src.zip/download#
+source_filename = libttf-1.4-src.zip
+source_hash = 533341269bdded2619622222e79af8af066b98b9183dbafda663907daabef309
+
+patch_directory = freetype1
+
+[provide]
+freetype1 = freetype_dep
```

Comment:
The value must match the name of the variable used in `meson.build` (currently `ttf_dep`).

Reply:
Fixed and `git push`ed. 

How about the license field in `project()` ? It is a custom license with a list of active Patents.

Reply:
Needs a rebase against master.